codecov_cli/parsers/junit.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from datetime import datetime, timedelta
+from typing import List
+
+from lxml import etree
+
+from codecov_cli.parsers.base import Parser, ParsingError, Test, Testrun
+
+logger = logging.getLogger("codecovcli")
+
+
+class JUnitXMLParser(Parser):
+    def __init__(self):
+        self._parser = etree.XMLParser(recover=True, resolve_entities=False)
+
+    def parse(self, file_content) -> List[Testrun]:
+        processed = self._parse_xml(file_content)
+        if processed is None or len(processed) == 0:
+            raise ParsingError("Error parsing XML file")
+
+        testsuites = [
```

Comment:
You are calling this `testsuites` but the type name is `TestRun`. I think you should pick one and stick to it. Whatever makes the most sense in the context.